compiler/common-artifacts/CMakeLists.txt

```diff
@@ -1,14 +1,14 @@
 #[[ Generate common python virtual enviornment ]]
-find_package(PythonInterp 3 QUIET)
-find_package(PythonLibs 3 QUIET)
+find_package(PythonInterp 3.8 QUIET)
+find_package(PythonLibs 3.8 QUIET)
```

Comment:
@mhs4670go , `PythonLibs` seems not used, can we remove this? (I'll remove in another PR if so) If used, could you please point me where?

Reply:
@seanshpark As [document](https://cmake.org/cmake/help/v3.10/module/FindPythonLibs.html) said, `PythonLibs` finds python libraries. I don't know what the libraries exactly are but I think it is `libpython*.so` things. Maybe default python libraries. I needs some test if whether it is really being used by removing above line and building it. Please let me take some time.

FYI, `PythonLibs` has been [deprecated](https://cmake.org/cmake/help/latest/module/FindPythonLibs.html) since cmake 3.12. We'll need to update it later.

Reply:
OK, let's go with this for a while :)